src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__link"
+        href="images/logo.png"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
```

Comment:
Each BEM element that is used for styling should have a corresponding block. 'is-active' seems to be a modifier but it's not clear which block or element it is modifying. Ensure that it follows the BEM naming convention.